hazelcast/src/main/java/com/hazelcast/monitor/NearCacheStats.java

```diff
@@ -19,44 +19,58 @@
 public interface NearCacheStats extends LocalInstanceStats {
 
     /**
-     * Returns the creation time of this NearCache on this member
+     * Returns the creation time of this Near Cache on this member.
      *
-     * @return creation time of this NearCache on this member
+     * @return creation time of this Near Cache on this member.
      */
     long getCreationTime();
 
     /**
-     * Returns the number of entries owned by this member.
+     * Returns the number of Near Cache entries owned by this member.
```

Comment:
That's a good point. Is it okay if I fix it in the pending cleanup PR? I have to rebase that one anyway.